crates/ckb-did-plc-utils/src/operation.rs

```diff
@@ -240,9 +210,13 @@ impl Operation {
             Err(Error::VerifySignatureFailed)
         }
     }
+
     pub(crate) fn generate_cid(&self) -> Result<String, Error> {
         let mut writer = BufWriter::new(Vec::new());
-        self.raw
+
+        // Convert Vec<(Value, Value)> back to Value::Map for encoding
+        let map_value = Value::Map(self.raw.clone());
```

Comment:
we may use core type to avoid clone:
```suggestion
        let map_value = cbor4ii::core::types::Map(self.raw.as_slice());
```